src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2192,7 +2192,7 @@ private static bool RequiredFunctionType<TMember>(MemberResolutionResult<TMember
             static bool isAcceptableRefMismatch(RefKind refKind, bool isInterpolatedStringHandlerConversion)
                 => refKind switch
                 {
-                    RefKind.In => true,
+                    RefKind.In or RefKind.RefReadOnlyParameter => true,
```

Comment:
Spec: Should the spec mention that we'll have a preference of by-value over `ref readonly` overload?